ADApp/pluginSrc/NDPluginDriver.cpp

```diff
@@ -357,7 +361,11 @@ asynStatus NDPluginDriver::writeInt32(asynUser *pasynUser, epicsInt32 value)
         this->lock();
     } else if (function == NDPluginDriverQueueSize) {
         newQueueSize_ = value;
-     } else {
+    } else if (function == NDPluginDriverProcessPlugin) {
+        if (pInputArray_) {
+            driverCallback(pasynUserSelf, pInputArray_);
+        }
```

Comment:
I have now printed an error message and returned an error status.  I improved the error handling in general in the NDPluginDriver::writeInt32() method.